### PR TITLE
[SIG-4885] Start removing jenkins references out of signals-frontend.

### PR DESCRIPTION
**Pull request by OscarBakker**
Ticket: [SIG-4885](https://datapunt.atlassian.net/browse/SIG-4885)

## Signalen

Before opening a pull request, please ensure:

- [x] Double-check your branch is based on `develop` and targets `develop`
- [x] Pull request has tests (we are going for 100% coverage!)
- [x] Code is well-commented, linted and follows project conventions
- [x] Committed source code is headed by the correct SPDX license expression

Be kind to code reviewers, please try to keep pull requests as small and focused as possible :)

### CODE REVIEW

**Comment by OscarBakker** [comment]
@bartjkdp He bart, can you maybe here add comments how the changes should be made. In the deploy-setup.md there are a lot of references to Jenkins. I tried changing some of them but its definitely more your territory:)

It would be nice to have a document that reflects the current setup:)

File: docs/adr/0005-deploy-setup.md [hunk 1]
```diff
@@ -15,25 +15,25 @@ Previously, the application was split into [multiple parts](./0004-multi-tenant-
 - [signals-weesp](https://github.com/Amsterdam/signals-weesp) and
 - [signals-amsterdamsebos](https://github.com/Amsterdam/signals-amsterdamsebos) (does not exist)
 
-The first repository in the list contains the functionality and the others contain domain specific configuration. At deployment, a Docker image of the first repository is created and pushed to the Docker repository by a Jenkins job. Note that only tags and the `develop` branch can be deployed.
+The first repository in the list contains the functionality and the others contain domain specific configuration. At deployment, a Docker image of the first repository is created and pushed to the Docker repository by a Azure Container registery job. Note that only tags and the `develop` branch can be deployed.
```
Comment (bartjkdp) [inline docs/adr/0005-deploy-setup.md:18]:
```suggestion
The first repository in the list contains the functionality and the others contain domain specific configuration. At deployment, a Docker image of the first repository is created and pushed to the Docker repository by a Github actions job. Note that only tags can be deployed.
```

File: docs/adr/0005-deploy-setup.md [hunk 2]
```diff
@@ -15,25 +15,25 @@ Previously, the application was split into [multiple parts](./0004-multi-tenant-
...
 
 ## Decision
 
 All domain specific repositories will be combined into [a single repository](https://github.com/Amsterdam/signalen). This repository will contain a separate 'package' (or whatever you want to call it) that contains domain specific configuration.
 
-The [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository will remain the base repository and will have tagged releases, as is currently the case. The `Jenkinsfile` will be moved to [Amsterdam/signalen](https://github.com/Amsterdam/signalen).
+The [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository will remain the base repository and will have tagged releases, as is currently the case.
 
-Each separate package will still have their own `Dockerfile`. The [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository's `master` branch will be tagged, not individual 'packages'.
+Each separate package will have their own `Dockerfile`. The [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository's `master` branch will be tagged, not individual 'packages'.
```
Comment (bartjkdp) [inline docs/adr/0005-deploy-setup.md:36]: You can remove this, this is now part of Azure DevOps.

File: docs/adr/0005-deploy-setup.md [hunk 3]
```diff
@@ -45,11 +45,11 @@ Schema validation at Docker runtime should be taken out.
 
 ### Deployment
 
-A single `Jenkinsfile` in the [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository will tie the [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository and the separate 'packages' together. This will prevent having race conditions.
+~~A single `Jenkinsfile` in the [Amsterdam/signalen](https://github.com/Amsterdam/signalen) repository will tie the [Amsterdam/signals-frontend](https://github.com/Amsterdam/signals-frontend) repository and the separate 'packages' together. This will prevent having race conditions.~~
```
Comment (OscarBakker) [inline docs/adr/0005-deploy-setup.md:48]: and these strikethrough lines @bartjkdp

Comment (bartjkdp) [reply docs/adr/0005-deploy-setup.md:48]: Yes, please remove this 👍  also the repository Amsterdam/signalen can be deleted.

Comment (OscarBakker) [reply docs/adr/0005-deploy-setup.md]: heb die repo verwijderd @bartjkdp!

**Review by SireeKoolenWijkstra — APPROVED** [review]
lgtm

**Pull request merged**